src/platform/modulesmanager/inc/ManagementModule.h

```diff
@@ -11,12 +11,12 @@
 #include <Mmi.h>
 
 // MMI function definitions
-using MmiGetInfo_ptr = int (*)(const char *, MMI_JSON_STRING *, int *);
-using MmiFree_ptr = void (*)(MMI_JSON_STRING);
-using MmiOpen_ptr = MMI_HANDLE (*)(const char *, const unsigned int);
-using MmiSet_ptr = int (*)(MMI_HANDLE, const char *, const char *, const MMI_JSON_STRING, const int);
-using MmiGet_ptr = int (*)(MMI_HANDLE, const char *, const char *, const MMI_JSON_STRING *, int *);
-using MmiClose_ptr = void (*)(MMI_HANDLE);
+using Mmi_GetInfo = int (*)(const char *, MMI_JSON_STRING *, int *);
+using Mmi_Free = void (*)(MMI_JSON_STRING);
```

Comment:
No space between type and * please. Here are elsewhere.

Instead of:

Foo *,  Blah *, 

Do:

Foo*, Blah*